### PR TITLE
Footer functionality

### PR DESCRIPTION
**Pull request by jordibares**
Hi Alex, this is my implementation of the footer based on other themes, I would consider adding some social media buttons and RSS feed but it is a bit too much for me right now, in any case, the structure seems to me ok.
Please let me know if it makes sense at all.

Thanks again

### CODE REVIEW

**Comment by alex-shpak** [comment]
#38: for reference

File: layouts/partials/docs/html-head.html [hunk 1]
```diff
@@ -2,11 +2,30 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <title>{{- template "title" . }} | {{ .Site.Title -}}</title>
 
+<!-- Font Awesome 5.8/1 no need for local files using all.css so all icons are available -->
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```
[the remark this answers is not in the document]
Comment (jordibares) [reply layouts/partials/docs/html-head.html]: Got it. I was exploring having social media icons and left it in place but I see what you mean.

File: static/custom.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@charset "UTF-8";
```
[the remark this answers is not in the document]
Comment (jordibares) [reply static/custom.css]: Got it. I will let you asses the scss but I must confess I am not that fond of it for tiny things like this one... anyway, you decide.

File: layouts/partials/docs/html-head.html [hunk 3]
```diff
@@ -2,11 +2,30 @@
...
+<!-- Theme stylesheet, if possible do not edit this stylesheet -->
 <link rel="stylesheet" href="{{ "normalize.min.css" | relURL }}">
 {{ $styles := resources.Get "book.scss" | resources.ToCSS | resources.Minify | resources.Fingerprint }}
 <link rel="stylesheet" href="{{ $styles.RelPermalink }}">
 
+<!-- Custom stylesheet - for your changes -->
+<link rel="stylesheet"  href="{{ "custom.css" | relURL }}?v={{ now.Unix }}" type='text/css' media='all'>
+
+<!-- Favicon -->
+<link rel="shortcut icon" href="{{ "img/favicon.ico" | absURL }}" type="image/x-icon">
```
[the remark this answers is not in the document]
Comment (jordibares) [reply layouts/partials/docs/html-head.html]: Great idea.. will do that.

File: layouts/partials/docs/footer.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!-- Copyright notice -->
```
[the remark this answers is not in the document]
Comment (jordibares) [reply layouts/partials/docs/footer.html]: Just to clarify, my thinking was to allow users to add their copyright notices there but using the yaml parameter, failing to do so will return your copyright notice. But all of it is be default turned to false to maintain your original setup.
If you feel like this is too intrusive, by all means do what you think is right, I want to pepper your theme with stuff you don't want.
:-)

**Comment by jordibares** [comment]
Done, all ready for you.

**Comment by jordibares** [comment]
Also updated the README.md

File: README.md [hunk 5]
```diff
@@ -162,8 +168,10 @@ There are few empty partials you can override in `layouts/partials/`
 | --                                               | --                                      |
 | `layouts/partials/docs/inject/head.html`         | Before closing `<head>` tag             |
 | `layouts/partials/docs/inject/body.html`         | Before closing `<body>` tag             |
+| `layouts/partials/docs/inject/footer.html`       | Before closing `<body>` tag             |
```
Comment (alex-shpak) [inline README.md]: Looks like a duplicate :)

File: layouts/partials/docs/html-head.html [hunk 6]
```diff
@@ -2,11 +2,28 @@
...
+<!-- Fonts for the theme -->
 <link href="https://fonts.googleapis.com/css?family=Oxygen|Oxygen+Mono:300,400,700" rel="stylesheet">
+
+<!-- Theme stylesheet, if possible do not edit this stylesheet -->
 <link rel="stylesheet" href="{{ "normalize.min.css" | relURL }}">
 {{ $styles := resources.Get "book.scss" | resources.ToCSS | resources.Minify | resources.Fingerprint }}
 <link rel="stylesheet" href="{{ $styles.RelPermalink }}">
 
+<!-- Custom stylesheet - for your changes -->
+{{ if (fileExists "static/img/banner.jpg") -}}
```
Comment (alex-shpak) [inline layouts/partials/docs/html-head.html]: Wrong files checked :)

File: exampleSite/config.yml [hunk 7]
```diff
@@ -46,4 +46,10 @@ params:
   # Configure the date format used on the pages
   # - In git information
   # - In blog posts
-  BookDateFormat: "Jan 2, 2006"
\ No newline at end of file
+  BookDateFormat: "Jan 2, 2006"
+
+  # (Optional, default false) Show or hide site footer
+  BookShowFooter: false
```
Comment (alex-shpak) [inline exampleSite/config.yml]: I think I would like to remove these two options, just to keep it minimalistic, and make `docs/inject/footer.html` empty, so people can override.

File: README.md (hunk 5 above)
Comment (jordibares) [reply README.md]: Opss... thanks

File: layouts/partials/docs/html-head.html (hunk 6 above)
Comment (jordibares) [reply layouts/partials/docs/html-head.html]: fixed

File: exampleSite/config.yml (hunk 7 above)
Comment (jordibares) [reply exampleSite/config.yml]: got it.

**Comment by jordibares** [comment]
Done

**Comment by jordibares** [comment]
Is there anything left to do? I think i am finished on my end. cheers

**Comment by alex-shpak** [comment]
Sorry, was traveling.
Looks good!

**Pull request merged**